### PR TITLE
Reduce dependency for ``brctl``

### PR DESCRIPTION
**Pull request by unakatsuo**
[iproute2 cheat sheet](http://baturin.org/docs/iproute2/#Create+a+bridge+interface) gives examples to play with Linux bridge without ``brctl``.

### CODE REVIEW

**Comment by Metallion** [comment]
Removing dependencies is awesome and this code is ok to be merged but I'm a little confused by the fact that you've removed brctl from some parts of the multibox code but not others.

Would it be possible to remove them  from here too?

```
openvdc/ci/citest/acceptance-test/multibox) > git grep -n brctl

build.sh:37:check_dep "brctl"
ind-steps/step-box/boot.sh:66:            sudo brctl show ${nic[2]#*=} | grep -wq ${nic[0]#*=}
ind-steps/step-box/boot.sh:69:            sudo brctl addif ${nic[2]#*=} ${nic[0]#*=}
ind-steps/step-box/common.source:39:              ! (brctl show ${nic[2]#*=} | grep -wq ${nic[0]#*=})
ind-steps/step-box/common.source:41:              sudo brctl delif ${nic[2]#*=} ${nic[0]#*=}
```

Just for the record, before this pull request I didn't even know it was possible to use the ip command to handle bridges. Good stuff. :)

**Comment by unakatsuo** [comment]
The main objective of this PR is to avoid the error below:

https://ci.openvdc.org/blue/rest/organizations/jenkins/pipelines/citest/branches/zk-sync/runs/5/nodes/36/log

```
=== RUN   TestLXCInstance_NICx2
--- FAIL: TestLXCInstance_NICx2 (15.95s)
	02_start_and_terminate_lxc_instance_test.go:57: eth1
	02_start_and_terminate_lxc_instance_test.go:59: Interface i-0000000001_00 is not attached
	02_start_and_terminate_lxc_instance_test.go:66: i-0000000001_00
	02_start_and_terminate_lxc_instance_test.go:68: Interface i-0000000001_01 is not attached
```

brctl seems to return the interface names in random order sometime. And it was hard to parse its output from ``brctl show``.

**Comment by Metallion** [comment]
Ok sounds good. :) Thanks for the clarification.

**Review by Metallion — APPROVED** [review]
:+1:

**Comment by unakatsuo** [comment]
Someone else can remove remaining brctl.

**Comment by Metallion** [comment]
home: can't someone else do it?
